js/components/Privacy.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * Implements the Privacy rules.
+ *
+ * @providesModule Provacy
+ * @flow
+ */
+
+'use strict'
+
+const React = require('React')
+
+import { StyleSheet, View, Text } from 'react-native'
+
+import { CommonStyles } from './styles'
+
+import I18n from '../i18n'
+
+const styles = StyleSheet.create({
+  content: {
+    backgroundColor: '#D8D8D8',
+    flex: 1,
+    paddingLeft: 24,
+    paddingRight: 24
+  },
+  closeModal: {
+    fontSize: 30,
+    textAlign: 'right',
+    paddingTop: 40,
+    fontFamily: 'Titillium Web',
+    color: '#17324D'
+  },
+  title: {
+    paddingTop: 50
+  },
+  mainText: {
+    paddingTop: 15
+  }
+})
+
+export class Privacy extends React.Component {
+  props: {
+    closeModal: () => void
+  }
+  _handleBack() {
+    this.props.closeModal()
+  }
+  render() {
+    return (
+      <View style={styles.content}>
+        <Text
+          style={styles.closeModal}
+          onPress={() => {
+            this._handleBack()
+          }}
+        >
+          {' '}
+          X{' '}
```

Comment:
I believe this should be a [transparent icon button](https://docs.nativebase.io/Components.html#button-icon-headref).

Also it would be nice to decouple the modal frame (with close button) from the actual content, so that we can reuse the modal frame in other parts of the application

I suggest we use a library component like [this one](https://github.com/maxs15/react-native-modalbox) or [this one](https://github.com/react-native-community/react-native-modal) instead of rolling out our own as they provide more features out of the box (i.e. swipe to close or backdrop press to close)